packages/react-shopify-app-route-propagator/README.md

```diff
@@ -3,16 +3,54 @@
 [![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-shopify-app-route-propagator.svg)](https://badge.fury.io/js/%40shopify%2Freact-shopify-app-route-propagator) ![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/react-shopify-app-route-propagator.svg)
 
-`<ShopifyRoutePropagator />` is a simple component to synchronize a Shopify embedded app's client side routing with the outer iframe host.
+`<ShopifyRoutePropagator />` is a simple component to synchronize a Shopify embedded app's client side routing with the outer iframe host. It assume the embedded app is using [Shopify's App Bridge Library](https://help.shopify.com/en/api/embedded-apps/app-bridge)
 
 The component is quite small and can be used with any routing solution.
 
 ## Installation
 
 ```bash
+$ yarn add @shopify/app-bridge
```

Comment:
Would specify only if you do not have it already otherwise people mindlessly c/p.

Reply:
if they have it already, yarn will not install it again.